### PR TITLE
modify native token symbol

### PR DESCRIPTION
**Pull request by zjb0807**
Use `COIN` to modify the native token symbol.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -37,7 +37,10 @@ RUN mix do deps.get, local.rebar --force, deps.compile
 ADD . .
 
 ARG COIN
-RUN if [ "$COIN" != "" ]; then sed -i s/"POA"/"${COIN}"/g apps/block_scout_web/priv/gettext/en/LC_MESSAGES/default.po; fi
+RUN if [ "$COIN" != "" ]; then \
```
Comment (vbaranov) [inline docker/Dockerfile:40]: As per https://docs.blockscout.com/for-users/faqs/how-can-i-customize-the-coin-symbol, "Ether" in .pot file should be modified as well

**Review by vbaranov — COMMENTED** [review]
@zjb0807 `make start` gives me '$COIN' instead of coin name

Screenshot 2022-03-09 at 14 42 54

**Comment by vbaranov** [comment]
@zjb0807 what do you think on this implementation https://github.com/blockscout/blockscout/pull/3814/files ?

**Comment by zjb0807** [comment]
> @zjb0807 what do you think on this implementation https://github.com/blockscout/blockscout/pull/3814/files ?

Our purpose is the same. but my code is more concise.

**Review by vbaranov — APPROVED** [review]
Looks fine

**Pull request merged**